compiler_opt/rl/compilation_runner.py

```diff
@@ -275,6 +285,46 @@ def enable(self):
   def cancel_all_work(self):
     self._cancellation_manager.kill_all_processes()
 
+  @staticmethod
+  def get_rewards(result: Dict) -> List[float]:
+    return [v[1] for v in result.values()]
+
+  def collect_results(
```

Comment:
i think it is over-engineering to refactor the collect_data function with the collect_results function, the options in the collect_results here are too complicated and the return type is also confusing, the name of collect_results and collect_data is also confusing.

I recommend to just write a function for your evaluation purpose, in this way the function args and return types will be much clearer

Reply:
that's sounds reasonable. I'll just make `_compile_fn` public scoped, that's essentially all I need anyway